### ISSUE TITLE
Flip tilde ligation

### ISSUE THREAD

**Issue by akarzim** [be5invis/Iosevka#959]
- [x] A programming language requires this ligation. Provide the information below.
- [x] This ligation is implemented in other programming fonts. Provide samples below.

------

The tilde ligature looks odd because it starts with a curve from top to bottom, whereas the tilde character starts with a curve from bottom to top. 

![tilde](https://user-images.githubusercontent.com/130560/115142561-c054ca00-a042-11eb-88a0-b4f9e7259fd0.png)

I propose to flip this ligature to end with this rendering:

![tilde-reversed](https://user-images.githubusercontent.com/130560/115142565-c5b21480-a042-11eb-8c7c-76bd9df85194.png)

An even better rendering would be to start **and end** the curve as the tilde character does.

### Examples from other fonts

[Fira Code](https://github.com/tonsky/FiraCode/)
<img width="804" alt="image" src="https://user-images.githubusercontent.com/130560/115142671-4d981e80-a043-11eb-992f-99a87ec4bdd7.png">
[Victor Mono](https://rubjo.github.io/victor-mono/)
<img width="1469" alt="image" src="https://user-images.githubusercontent.com/130560/115142700-715b6480-a043-11eb-9329-497b0909cacd.png">

**Comment by be5invis** [comment]
"start and end the curve as the tilde character does" will create broken shapes. Adjust starting should be enough.

**Comment by lorenzogatti** [comment]
The boundaries between the left, middle and right section of the extended tilde could fall on the inflection points rather than on the upward or downward bumps. The middle section would have a full period, shifted by one fourth period compared to the current design, and the left and right sections could have a full wave (slightly distorted and/or shortened at the outside end), i.e. one fourth of a period of the wave more, so that they end at the inflection point too and the direction remains consistent. 
In other words, the right section should be like the left section rotated 180* rather than like the left section reflected and the middle section should have odd rather than even symmetry.
